src/icloudpd/base.py

```diff
@@ -769,158 +769,148 @@ def core(
     # Access to the selected library. Defaults to the primary photos object.
     library_object = icloud.photos
 
-    if list_libraries:
-        libraries_dict = icloud.photos.libraries
-        library_names = libraries_dict.keys()
-        print(*library_names, sep="\n")
+    # TODO: selecting library not supported anymore! Need to adjust USAGE, docs, etc
```

Comment:
This is another significant change. The latest `PyiCloud` does not support `libraries`, so we have to use the default `photos.all`. I'll need to update any CLI args/USAGE docs to remove the ability to select libraries, as it's not supported anymore... Or so it would seem.

Reply:
This is actually a case where the `pyicloud_ipd` fork added functionality that was never implemented in the original 'pyicloud' project. There is PR picklepete/pyicloud#407 to merge this into the 'pyicloud' project, but as that seems to be a bit of a dead project now.  There is a bit of cross merging that needs to occur to truly get the latest features on both sides.  I wouldn't necessarily remove this feature from this project.

Reply:
I wondered if it was a patch added after the fact. 

The intention was not to leave out all of the things I removed (domain selection, library selection, etc), the intention was to address the bug first, and then figure out what we can do about the rest once the project was in a more working state again.